scripts/build-circt.sh

```diff
@@ -43,12 +48,15 @@ while [[ "$#" -ge 1 ]] ; do
 	esac
 done
 
-git clone https://github.com/EECS-NTNU/circt.git ${CIRCT_BUILD}
-cd ${CIRCT_BUILD}
+if [ ! -d "$CIRCT_BUILD/circt.git" ] ;
+then
+	git clone https://github.com/EECS-NTNU/circt.git ${CIRCT_BUILD}/circt.git
```

Comment:
I have not tested it, but what happens if you try to `git clone` something into a repository that is already there? In other words, is it possible to remove the `if [.....` statement and let `git clone` just do its thing?

Reply:
git clone fails if the directory already exists.